### PR TITLE
[eclipse/xtext#1686] prepared R2020-03 release target platform

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1686] prepared R2020-03 release target platform
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -86,8 +86,10 @@ spec:
         
         script {
           if (params.target_platform == 'latest') {
+            currentBuild.displayName = "#${BUILD_NUMBER}(4.16)"
+          } else if (params.target_platform == 'r202003') {
             currentBuild.displayName = "#${BUILD_NUMBER}(4.15)"
-          } else if (params.target_platform == 'r201912') {
+          }  else if (params.target_platform == 'r201912') {
```
Comment (miklossy) [inline Jenkinsfile]: Please revert this change.

Comment (cdietrich) [reply Jenkinsfile]: done
